dotnet-developer-projects.md

```diff
@@ -92,6 +92,7 @@ Please sort projects alphabetically and provide a one-line description. GitHub/C
   * [MassTransit](https://github.com/MassTransit/MassTransit) - A distributed application framework for RabbitMQ and Azure Service Bus.
   * [NServiceBus](https://github.com/Particular/NServiceBus) - The most popular service bus for .NET
   * [RestBus](https://github.com/tenor/RestBus) - Easy Asynchronous Messaging and Queueing for .NET
+  * [surging](https://github.com/dotnetcore/surging) -The distributed micro service framework based on.NET Core provides high-performance RPC Communications
```

Comment:
on.NET --> on .NET

Reply:
Thank you for reviews, changes to Libraries